diff --git a/src/EventSubscriber/SentryEventSubscriber.php b/src/EventSubscriber/SentryEventSubscriber.php
index 2c1b7f6..7a3d79b 100644
--- a/src/EventSubscriber/SentryEventSubscriber.php
+++ b/src/EventSubscriber/SentryEventSubscriber.php
@@ -8,6 +8,7 @@ use Symfony\Component\DependencyInjection\ContainerAwareInterface;
 use Symfony\Component\DependencyInjection\ContainerAwareTrait;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\HttpKernel\Event\ExceptionEvent;
+use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
 use Symfony\Component\HttpKernel\KernelEvents;
 
 /**
@@ -50,10 +51,10 @@ class SentryEventSubscriber implements EventSubscriberInterface, ContainerAwareI
   /**
    * Log all exceptions.
    *
-   * @param \Symfony\Component\HttpKernel\Event\ExceptionEvent $event
+   * @param \Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent $event
    *   The event to process.
    */
-  public function onException(ExceptionEvent $event) {
+  public function onException(GetResponseForExceptionEvent $event) {
     $this->sentry->log($event);
   }
 
